src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/FromFhirToEdifact.java

```diff
@@ -0,0 +1,32 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Segment;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FromFhirToEdifact {
+    //TODO fix mapper order if needed
+    //TODO use dependency injection as a list
+    private final List<FromFhirToEdifactMapper<?>> MAPPERS = Arrays.asList(
```

Comment:
Mappers will need to be parametrized as we won't be mapping all of the stuff (different transactions need different data). For now it might stay as is, but next GP->HA transaction will need to address that.

Reply:
Intellij says this file is unused. Please remove it.

Reply:
FromFhirToEdifact removed